script.js

```diff
@@ -171,7 +201,7 @@ document.addEventListener("DOMContentLoaded", () => {
   displayMovies(movies);
```

Comment:
This piece of code throws an error:
<img width="565" alt="image" src="https://github.com/user-attachments/assets/2d3fa666-dcd2-4da6-8761-ce359c9060ff">

Reply:
Hi, could you please check my code? Now it is not showing any error in console.

Reply:
Nice!